mujoco_warp/_src/math.py

```diff
@@ -217,6 +217,14 @@ def make_frame(a: wp.vec3):
   # fmt: on
 
 
+@wp.func
+def normalize_with_norm(x: wp.vec2):
```

Comment:
can you just define a single `normalize_with_norm` but it takes `Any`?

It looks like wp.func allows any: https://github.com/NVIDIA/warp/blob/73d5a063505e64b35d90894f8f452a00fa82c897/warp/math.py#L57

Reply:
done